modules/ocf_printhost/files/enforcer-pc

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gs -o /dev/null -sDEVICE=bbox $1 2>&1 | grep HiResBoundingBox | wc -l
```

Comment:
OK, my bad on the bash script, I didn't really think about it because I meant it as a workaround since using pipes in a Python subprocess requires some effort.

I think it makes sense to leave it as a separate script, though. As you suggested, testing would be a valuable benefit, for one thing. It would also be nice to find out a faster way to do the page count, but it probably won't matter since we cap documents to 30 pages. Still, it's way faster to page count PDFs with GhostScript (e.g. can count a 398 page PDF in < 1s while it takes 3s to count a 5 page document with this script. Blame grep).